device/tt_soc_descriptor.cpp

```diff
@@ -314,6 +315,12 @@ void tt_SocDescriptor::get_cores_and_grid_size_from_coordinate_manager() {
     harvested_cores_map.insert({CoreType::ETH, coordinate_manager->get_harvested_cores(CoreType::ETH)});
     harvested_grid_size_map.insert({CoreType::ETH, coordinate_manager->get_harvested_grid_size(CoreType::ETH)});
 
+    harvested_cores_map.insert({CoreType::ARC, coordinate_manager->get_harvested_cores(CoreType::ARC)});
```

Comment:
So for our physical coord system, if we look at core which is a harvested tensix, will it have CoreType::TENSIX or CoreType::HARVESTED? What is ::HARVESTED used for exactly?